lib/serializer.js

```diff
@@ -62,17 +63,20 @@ Serializer.prototype._concatKeyVal = function(v, k) {
 };
 
 Serializer.prototype._serializeValue = function(v, k) {
-    if (_.isBoolean(v)) {
+    if (values.isBoolean(v)) {
         return [k, (v ? 't' : 'f')];
     } else
-    if (_.isString(v)) {
+    if (values.isString(v)) {
         return [k, '"' + this._escapeString(v) + '"'];
     } else
     if (this._isInt(v, k)) {
         return [k, Math.floor(v) + 'i'];
     } else
-    if (v instanceof JuttleMoment) {
+    if (values.isDate(v) || values.isDuration(v)) {
         return [k, v.unixms()];
+    } else
+    if (values.isObject(v) || values.isArray(v)) {
+        return [k, '"' + this._escapeString(JSON.stringify(v)) + '"'];
```

Comment:
That's a good question. TBH, I'm not sure - is it better to silently convert these fields, or reject the whole point just because it contains an object/array field? IMHO this depends on an unknown - how often the array/object fields will occur in practice/how often this feature will be used.